src/Microsoft.FileFormats/PE/PEStructures.cs

```diff
@@ -305,4 +306,19 @@ public class VsVersionInfo  : TStruct
     }
 
     #endregion
+
+    /// <summary>
+    /// VS_PDBCHECKSUM struct
```

Comment:
How about: `Pair of a checksum algorithm name (ex: "SHA256") and the bytes of the checksum.`